### DISCUSSION TITLE
这个平台能否配置天地图组件？如果能，如何配置一个天地图组件到平台中去？

### DISCUSSION THREAD

**Discussion by xmh0604** [opentiny/tiny-engine#107] [Q&A]
我们公司自己封装了个天地图组件，按照官网给的ELinput例子配置后，一直显示区块加载中，也没有报错，配置了bundle.json与appinfo.json文件，目前找不到问题在哪，是否可以讨论一下？

**Comment by wenmine** [comment]
目前我们导入第三方组件库的功能，更多支持的是使用vue3开发的，比较规则、标准的组件库，我理解你说的这个组件更多的是自定义的一个区块？目前我们区块的能力因为依赖于构建、发布，这块的解决方案需要等到后端开源后，才会完整的进行提供
